docs/server/source/cloud-deployment-templates/node-on-kubernetes.rst

```diff
@@ -42,8 +63,254 @@ then try adding ``--ssh-key-file ~/.ssh/<name>``
 to the above command (i.e. the path to the private key).
 
 
-Step 3: Create Storage Classes
-------------------------------
+Step 3: Connect to the Cluster UI - (optional)
+----------------------------------------------
+
+* Get the kubectl context for this cluster using ``kubectl config view``.
+ 
+* For the above commands, the context would be ``k8s-bdb-test-cluster-0``.
+ 
+  .. code:: bash
+
+     $ kubectl --context k8s-bdb-test-cluster-0 proxy -p 8001
+
+
+Step 4: Configure the Node
```

Comment:
I just noticed that there are two steps labelled "Step 4". I had an idea to save work: merge Step 1 and Step 2 into one step about kubectl, so you don't have to renumber all the other steps. :smiley: